SPECS/re2/re2.spec

```diff
@@ -4,7 +4,7 @@
 Summary:        C++ fast alternative to backtracking RE engines
 Name:           re2
 Version:        %{shortver}
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
this change doesn't hurt, but might not technically be needed.

Reply:
mostly just fyi

Reply:
Same as the comment with grpc